cmd/remove.go

```diff
@@ -66,7 +68,30 @@ func RunRemove() *cobra.Command {
 		}
 
 		if removeAll {
-			if err := qb.DeleteTorrents(ctx, []string{"all"}, deleteFiles); err != nil {
+			if removePaused {
+				pausedTorrents, err := qb.GetPausedTorrents(ctx)
```

Comment:
You can use the newly added method `GetTorrentsWithFilters` which takes a req object where you can filter by state.